firefox-ios/Client/Frontend/Browser/Tabs/State/TabsPanelState.swift

```diff
@@ -138,6 +142,25 @@ struct TabsPanelState: ScreenState, Equatable {
                                   isInactiveTabsExpanded: state.isInactiveTabsExpanded,
                                   toastType: type)
 
+        case TabPanelMiddlewareActionType.scrollToTab:
+            // FIXME Where is the best place for this mapping logic?
+            // FIXME Just realized that the TabDisplayView doesn't know how to take an index into `tabs` and convert that to
+            // an index into the subset of normal tabs (active/inactive) or the subset of private tabs.
```

Comment:
Where is the best place for this mapping logic? 🤔  Should this be in the middleware chunk that dispatches the `.scrollToTab` action?